apps/rewards/app/components/Form/Field/FormField.js

```diff
@@ -32,12 +32,20 @@ const FormField = ({
             *
           </Text>
         )}
+        {hint && (
+          <div
+            style={{
+              marginLeft: '0.3rem',
+              display: 'flex',
+              alignItems: 'center',
+            }}
+          >
```

Comment:
It doesn't look like `display` and `alignItems` have any effect here

Reply:
Oh, but they do. This is how the help bubbles look without them:
![Screenshot from 2019-10-31 14-15-13](https://user-images.githubusercontent.com/16065447/67978827-f735e680-fbe8-11e9-87e8-49667ea65b26.png)

And this is how they look with them:
![Screenshot from 2019-10-31 14-16-29](https://user-images.githubusercontent.com/16065447/67978881-13d21e80-fbe9-11e9-937f-2bd14b9b59d4.png)

Subtle, I agree. Without these CSS props, the bubble is a little too high:
![Screenshot from 2019-10-31 14-17-23](https://user-images.githubusercontent.com/16065447/67978988-63184f00-fbe9-11e9-8272-1b894200d9a6.png)

Whereas with the props:
![Screenshot from 2019-10-31 14-20-13](https://user-images.githubusercontent.com/16065447/67979092-978c0b00-fbe9-11e9-829f-e0bae87615ad.png)

Reply:
oh nice! I didn't notice at first, but I completely agree now that you've pointed it out.